libs/common/src/utils/mapDataToSchema.ts

```diff
@@ -0,0 +1,38 @@
+import { get, omit, castArray } from 'lodash';
+import { DocumentData } from '@google-cloud/firestore';
+import * as logs from '../logs';
+
+type ParamSchema = { [key: string]: any };
+
+const mapDataToSchemaWithoutLogging = (
```

Comment:
Where is this being used?

Reply:
It's used in the function below it that's actually exported. I didn't watch to have the try/catch at every recursive iteration of the function, so I wanted to wrap just the top-level of it in the try/catch.